src/strategies/infinityprotocol-liquidity-pools/examples.json

```
@@ -5,7 +5,9 @@
       "name": "infinityprotocol-liquidity-pools",
       "params": {
         "address": "0xd8a1734945b9ba38eb19a291b475e31f49e59877",
-        "symbol": "SHARD"
+        "symbol": "SHARD",
+        "decimal": 18,
```

the decimal option is never used in the strategy file
```suggestion
        "symbol": "SHARD",
```